chart/compass/values.yaml

```diff
@@ -34,15 +34,15 @@ global:
       version: "PR-913"
     provisioner:
       dir:
-      version: "PR-913"
+      version: "PR-953"
```

Comment:
You are not making any changes in the `components` directory, so the images will not be created. The same is true for KEB and Provisioner tests.

Reply:
True. Only charts are updated